### PR TITLE
new packages: wcslib and cfitsio

### PR DESCRIPTION
**Pull request by knyipab**
They are dependencies of astrometry.net (https://astrometry.net/doc/build.html) which I am working on to build a package.

### CODE REVIEW

**Comment by knyipab** [comment]
I am not so familiar with how termux build system works with makefile system, so I `cp` /src to /build. Grateful to know if this is the correct way to handle makefile not found in `termux_step_make`.

File: tur/wcslib/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+TERMUX_PKG_VERSION="8.2.2"
+TERMUX_PKG_SRCURL=http://www.atnf.csiro.au/people/mcalabre/WCS/wcslib-${TERMUX_PKG_VERSION}.tar.bz2
+TERMUX_PKG_SHA256=6298220ae817f4e5522643ac4c3da2623be70a3484b1a4f37060bee3e4bd7833
+TERMUX_PKG_DEPENDS="cfitsio"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+--with-cfitsioinc=${TERMUX_PREFIX}/include
+"
+
+termux_step_post_configure () {
+    cp -r -n "${TERMUX_PKG_SRCDIR}"/* "${TERMUX_PKG_BUILDDIR}/"
```
Comment (licy183) [inline tur/wcslib/build.sh]: You can set `TERMUX_PKG_BUILD_IN_SRC=true`

**Comment by licy183** [comment]
Thanks!

File: tur/wcslib/build.sh (hunk 1 above)
Comment (knyipab) [reply tur/wcslib/build.sh]: Thanks!

**Pull request merged**